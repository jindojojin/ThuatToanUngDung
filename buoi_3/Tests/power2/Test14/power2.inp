7
8 19920000 185620 1147000000 1 33554432 1073741824
