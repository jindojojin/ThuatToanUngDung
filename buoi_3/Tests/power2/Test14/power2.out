1 8 33554432 1073741824
